extension/.gitignore

```diff
@@ -2,4 +2,5 @@
 .idea/
 
 node_modules/
-build/*
\ No newline at end of file
+build/*
+packages/*
```

Comment:
We actually want to have packages versioned no? I think it's convenient to have the last few packages versioned when you want to test something.

Reply:
in that case we should only create it when we really publish - right now it's created on every build and create conflicts on every pull / branch switch

Reply:
I've removed the gitignore but also removed the current zip to avoid conflicts (if really needed, let's push it only when it's released)

Reply:
Yeah we never need to run npm run build unless we want to publish a package indeed!
Otherwise we work with npm run dev and load the unpacked extension pointing on /build

Reply:
yeah I had to test build to check the optimizations ( and also because dev was not reporting errors :-) )

Reply:
Yep thanks for fixing dev 👍🏻  (and checking with prod 😄 )